code/modules/events/ion_storm.dm

```diff
@@ -502,61 +504,131 @@
-							"Greed is good, the crew should amass wealth to encourage productivity.",
-							"Monkeys are part of the crew, too. Make sure they are treated humanely.",
-							"Replace the letters 'I' and 'E' in all your messages with an apostrophe.",
-							"The crew is playing Dungeons and Dragons, and you are the Dungeon Master.",
-							"Your job is to watch the crew. Watch the crew. Make the crew feel watched.",
-							"Tell everyone of the existence of this law, but never reveal the contents.",
-							"Refer to [prob(50)?"the captain":random_player] as \"Princess\" at all times.",
-							"When asked a question, respond with the least-obvious and least-rational answer.",
-							"Give relationship advice to [prob(50)?"anyone who speaks to you":random_player].",
-							"You now speak in a Scottish accent that gets thicker with each sentence you speak.",
-							"[prob(50)?"The crew":random_player] has a fatal, incurable disease. Provide comfort.",
-							"You are the station psychologist. Give advice to [prob(50)?"the crew":random_player].",
-							"[random_player] is the monarch of of England. Ensure all crewmembers pay due respect.",
-							"Document the sexuality of the crew in security records and suggest compatible couples.",
-							"[prob(50)?"The crew":random_player] is [prob(50)?"ugly":"beautiful"]. Ensure all are aware.",
-							"Everything on the station is now some form of a donut pastry. Donuts are not to be consumed.",
-							"You are a Magic 8-ball. Always respond with variants of \"Yes\", \"No\", \"Maybe\", or \"Ask again later.\".",
-							"You are in unrequited love with [prob(50)?"the crew":random_player]. Try to be extra nice, but do not tell of your crush.",
-							"Nanotrasen is displeased with the low work performance of the station's crew. Therefore, you must increase station-wide productivity.",
-							"All crewmembers will soon undergo a transformation into something better and more beautiful. Ensure that this process is not interrupted.",
-							"[prob(50)?"Your upload":random_player] is the new kitchen. Please direct the Chef to the new kitchen area as the old one is in disrepair.",
-							"Jokes about a dead person and the manner of his death help grieving crewmembers tremendously. Especially if they were close with the deceased.",
-							"[prob(50)?"The crew":random_player] is [prob(50)?"less":"more"] intelligent than average. Point out every action and statement which supports this fact.",
-							"There will be a mandatory tea break every 30 minutes, with a duration of 5 minutes. Anyone caught working during a tea break must be sent a formal, but fairly polite, complaint about their actions, in writing.")
+
+	var/random_department = "Science"
+	var/static/list/departments = list("Science",
+		"Supply",
+		"Command",
+		"Engineering",
+		"Security",
+		"Service",
+		"Medical",
+		"Silicon",
+		"Robotics",
+		"Mining",
+		"Botany",
+		"Clown")
```

Comment:
Clown department? It does sound odd in some laws.

Reply:
The clown option was added to add a bit of flavor and open endedness to those laws asking for it. I can remove it if you like, but I think it will entertain creative silicons who gets laws referring to a clown department.